src/main/java/org/broadinstitute/hellbender/utils/codecs/GENCODE/GencodeGtfGeneFeature.java

```diff
@@ -0,0 +1,121 @@
+    }
+
+    public static GencodeGtfFeature create(long lineNumber,
+                                           String chromosomeName,
+                                           AnnotationSource annotationSource,
+                                           FeatureType featureType,
+                                           int genomicStartLocation,
+                                           int genomicEndLocation,
+                                           GenomicStrand genomicStrand,
+                                           GenomicPhase genomicPhase,
+                                           String geneId,
+                                           String transcriptId,
+                                           GeneTranscriptType geneType,
+                                           GeneTranscriptStatus geneStatus,
+                                           String geneName,
+                                           GeneTranscriptType transcriptType,
+                                           GeneTranscriptStatus transcriptStatus,
+                                           String transcriptName,
+                                           int exonNumber,
+                                           String exonId,
+                                           LocusLevel locusLevel,
+                                           ArrayList<OptionalField<?>> optionalFields,
+                                           String anonymousOptionalFields) {
+
+        return new GencodeGtfGeneFeature(lineNumber, chromosomeName, annotationSource, featureType, genomicStartLocation, genomicEndLocation, genomicStrand, genomicPhase, geneId, transcriptId, geneType, geneStatus, geneName, transcriptType, transcriptStatus, transcriptName, exonNumber, exonId, locusLevel, optionalFields, anonymousOptionalFields);
+    }
+
+    // ================================================================================================
+
+    private ArrayList<GencodeGtfTranscriptFeature> transcripts = new ArrayList<>();
+
+    // ================================================================================================
+
+    public void addTranscript(GencodeGtfTranscriptFeature transcript) { transcripts.add(transcript); }
+
+    public ArrayList<GencodeGtfTranscriptFeature> getTranscripts() {
+        return transcripts;
+    }
+
+    @Override
+    protected List<GencodeGtfFeature> getAllFeatures() {
+        ArrayList<GencodeGtfFeature> list = new ArrayList<>();
+        list.add(this);
+
+        for ( GencodeGtfTranscriptFeature transcript : transcripts ) {
+            list.addAll(transcript.getAllFeatures());
+        }
+
+        return list;
+    }
+
+    @Override
+    public boolean equals(Object other) {
```

Comment:
Same comment here about overriding `hashCode()` as well, and ensuring consistency with `equals()`